sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_exporter.py

```diff
@@ -73,8 +103,36 @@ def export(
         :return: The result of the export.
         :rtype: ~opentelemetry.sdk.metrics.export.MetricExportResult
         """
-        # TODO
-        return MetricExportResult.SUCCESS
+        result = MetricExportResult.SUCCESS
+        base_monitoring_data_point = kwargs.get("base_monitoring_data_point")
+        if metrics_data is None or base_monitoring_data_point is None:
+            return result
+        data_points = _metric_to_quick_pulse_data_points(
+            metrics_data,
+            base_monitoring_data_point=base_monitoring_data_point,
+            documents=kwargs.get("documents"),
+        )
+
+        token = attach(set_value(_SUPPRESS_INSTRUMENTATION_KEY, True))
+        try:
+            post_response = self._client.post(  # type: ignore
+                monitoring_data_points=data_points,
+                ikey=self._instrumentation_key,
+                x_ms_qps_transmission_time=_ticks_since_dot_net_epoch(),
+                cls=Response,
+            )
+            if not post_response:
+                result = MetricExportResult.FAILURE
+            header = post_response._response_headers.get("x-ms-qps-subscribed")  # pylint: disable=protected-access
```

Comment:
This is a confusingly named header. Is there code if Java you got this from?

Reply:
It is defined in the live metrics [spec](https://eng.ms/docs/cloud-ai-platform/azure-edge-platform-aep/aep-health-standards/observability/application-insights/live-metrics-indexer-processing/onboarding/sdkapi) as the header name that will be returned.